bok_choy/browser.py

```diff
@@ -207,7 +207,7 @@ def browser_check_func():
             return False, None
 
     browser_instance = Promise(
-        browser_check_func, "Browser is instantiated successfully.", try_limit=3).fulfill()
+        browser_check_func, "Browser is instantiated successfully.", try_limit=3, timeout=60).fulfill()
```

Comment:
90 works for me.

I think I didn't really explain well what I was thinking in my prior comment. If I'm understanding correctly, as long as the value is > 3x30 for the timeout, it doesn't really matter what it is. The try_limit will be the one that kicks in anyhow.